OADP-5354: Backup is failing on validation PartiallyFailed (#86780)
openshift/openshift-docs: cloud_experts_tutorials/cloud-experts-deploy-api-data-protection.adoc

[default]
role_arn = ${ROLE_ARN}
web_identity_token_file = /var/run/secrets/openshift/serviceaccount/token

> region=<aws_region> # <1>

Author: @bhardesty - are you ok for me to make this change to your ROSA docs?

Reviewer: @anarnold97 LGTM; however, these cloud expert tutorials are maintained by the MOBB team. I would check with them first. @michaelryanmcneill do you know who can verify this doc change?

EOF
$ oc -n openshift-adp create secret generic cloud-credentials \
 --from-file=${SCRATCH}/credentials
----
+

> <1> <aws_region> is the AWS region to use for {sts-short} endpoint.

Reviewer 2: ```suggestion
<1> Replace `<aws_region>` with the AWS region to use for the {sts-first} endpoint.
```

> <1> <aws-region> is the AWS region in which the cloud resources has been created.

Author: We could go with something like this, if suitable:
```
AWS recommends using Regional AWS Security Token Service (AWS STS) endpoints instead of the global endpoint to reduce latency, build in redundancy, and increase session token validity. 
```

> <1> <aws-region> is the AWS region in which the cloud resources has been created.

Author: Is there anything you want us to add? 

Or do you want us to use the AWS content - ```A Regional endpoint is the URL of the entry point within a particular region for an AWS web service. AWS recommends using Regional AWS Security Token Service (AWS STS) endpoints instead of the global endpoint to reduce latency, build in redundancy, and increase session token validity.````

> <1> <aws-region> is the AWS region in which the cloud resources has been created.

Reviewer 3: ```suggestion
<1> <aws_region> is the AWS region in which the cloud resources have been created.
```

> <1> <aws_region> is the AWS region in which the cloud resources have been created.

Author: ```suggestion
<1> <aws_region> is the AWS region to use for STS endpoint.
```
